### DISCUSSION TITLE
Launching applications from the watchface

### DISCUSSION THREAD

**Discussion by mimimix** [zepp-health/zeppos-forum#88] [Q&A] [answered]
I think it will help to implement fast access to some applications.
Is it possible to get this feature in version 3 of the os? ☺️

**Comment by MyoungXUE** [comment, marked as answer]
Hello, sorry for the wait.

You can refer to the [`IMG_CLICK` widget](https://docs.zepp.com/docs/watchface/api/hmUI/widget/IMG_CLICK/) to jump to different applications by passing in the type parameter, but the `data_type` enumeration on the documentation is not perfect yet, I will add this part as soon as possible.

In Zepp OS 3.0, we are going to introduce editable jump zones for both system applications and Mini Programs, with the icon style of the application icon.
